app/src/main/res/values/strings.xml

```diff
@@ -1568,8 +1568,12 @@
     <string name="addresses_add_address">Add address</string>
     <!-- Title of the "Manage addresses" screen -->
     <string name="addresses_manage_addresses">Manage addresses</string>
-    <!-- The header for the full name of an address -->
-    <string name="addresses_full_name">Full Name</string>
```

Comment:
This string needs to be marked for removedIn="101" https://github.com/mozilla-mobile/fenix/wiki/Removing-strings#marking-an-unused-string-to-be-removed